.storybook/main.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    stories: ['../src/**/*.stories.[tj]s'],
+};
```

Comment:
@AbdoulNuru , please make a new line at the of file to remove the eslint error message.

Reply:
done @Moise1 thanks.